locales-src/onboarding.ftl

```diff
@@ -69,6 +73,15 @@ onboarding-benefit-privacy-title = True Privacy
 # transparent about how we use it."
 onboarding-benefit-privacy-text = Everything we do honors our Personal Data Promise: Take less. Keep it safe. No secrets.
 
+onboarding-benefit-sync-title = Sync
+onboarding-benefit-sync-text = Take your bookmarks, passwords, history, and more everywhere you use { -brand-product-name }.
+
+onboarding-benefit-monitor-title = Monitor
+onboarding-benefit-monitor-text = Get notified when your personal info is in a data breach.
+
+onboarding-benefit-lockwise-title = Lockwise
+onboarding-benefit-lockwise-text = Manage passwords that are protected and portable.
```

Comment:
Not l10n, but this sounds quite strange to me.

Reply:
Including @aaronrbenson @Left47 , looks like all language needs to pass through at least [one more review](https://bugzilla.mozilla.org/show_bug.cgi?id=1578528#c7)